client/views/helpers/chunks/question_div/question_div.scss

```diff
@@ -390,3 +390,24 @@ $popular-question-color: #f1c40f;
   font-weight: 600;
   color: #777!important;
 }
+
+.remote-highlight:hover, .remote-highlight > input:hover {
+  cursor: pointer;
+}
+
+.remote-highlight {
+  display: block;
+  padding-left: 15px;
+  text-indent: -15px;
+}
+
+.remote-highlight > input {
+  width: 13px;
+  height: 13px;
+  padding: 0;
+  margin:0;
+  vertical-align: bottom;
+  position: relative;
+  top: -1px;
+  *overflow: hidden;
```

Comment:
@singhsanket143 any good reason to have `*` here?

Reply:
@peter-hank I have removed this.